### PR TITLE
CP VC mac changes to release branch

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: tools/extra/fpgadiag/common.py [hunk 1]
```diff
@@ -35,23 +36,53 @@
...
-PATTERN = (r'.*(?P<segment>\w{4}):(?P<bus>\w{2}):'
+
+pattern = (r'.*(?P<segment>\w{4}):(?P<bus>\w{2}):'
            r'(?P<dev>\w{2})\.(?P<func>\d).*')
 
-BDF_PATTERN = re.compile(PATTERN)
+bdf_pattern = re.compile(pattern)
+
+ROOT_PATH = '/sys/class/fpga'
+char_dev = '/dev/char'
```
Comment (jelonanX) [inline tools/extra/fpgadiag/common.py]: s/char_dev/CHAR_DEV

File: tools/extra/fpgadiag/common.py [hunk 2]
```diff
@@ -174,18 +206,23 @@ def get_eth_group_info(self, eth_grps):
         info = {}
         for eth_grp in eth_grps:
             with open(eth_grp, 'r') as f:
-                node = os.path.join(CHAR_DEV, f.readline().strip())
+                node = os.path.join(char_dev, f.readline().strip())
```
Comment (jelonanX) [inline tools/extra/fpgadiag/common.py]: s/char_dev/CHAR_DEV

File: tools/extra/fpgadiag/common.py [hunk 3]
```diff
@@ -174,18 +206,23 @@ def get_eth_group_info(self, eth_grps):
...
             data = struct.pack(self.if_fmt, self.if_len, *[0] * 5)
             ret = self.ioctl(
                 os.path.join(
-                    CHAR_DEV,
+                    char_dev,
```
Comment (jelonanX) [inline tools/extra/fpgadiag/common.py]: s/char_dev/CHAR_DEV

File: tools/extra/fpgadiag/common.py [hunk 4]
```diff
@@ -35,23 +36,53 @@
...
-    return int(val, 16)
+def convert_argument_str2hex(args, arg_list):
+    if isinstance(arg_list, list):
+        for i in arg_list:
+            try:
+                if getattr(args, i) is not None:
+                    setattr(args, i, int(getattr(args, i), 16))
+            except BaseException:
+                exception_quit(
+                    'Invlid argument {}: {}'.format(
```
Comment (jelonanX) [inline tools/extra/fpgadiag/common.py]: s/Invlid/Invalid

File: tools/extra/fpgadiag/common.py [hunk 5]
```diff
@@ -35,23 +36,53 @@
 import fcntl
 import stat
 import struct
+import mmap
 
-PATTERN = (r'.*(?P<segment>\w{4}):(?P<bus>\w{2}):'
+
+pattern = (r'.*(?P<segment>\w{4}):(?P<bus>\w{2}):'
```
Comment (jelonanX) [inline tools/extra/fpgadiag/common.py]: uppercase MACROs

File: tools/extra/fpgadiag/common.py [hunk 6]
```diff
@@ -35,23 +36,53 @@
...
            r'(?P<dev>\w{2})\.(?P<func>\d).*')
 
-BDF_PATTERN = re.compile(PATTERN)
+bdf_pattern = re.compile(pattern)
```
Comment (jelonanX) [inline tools/extra/fpgadiag/common.py]: uppercase MACROs

File: tools/extra/fpgadiag/common.py [hunk 7]
```diff
@@ -66,13 +97,13 @@ def __init__(self, segment, bus, dev, func):
 
     def read_bdf(self, path):
         symlink = os.readlink(path)
-        m = BDF_PATTERN.match(symlink)
+        m = bdf_pattern.match(symlink)
```
Comment (jelonanX) [inline tools/extra/fpgadiag/common.py]: s/bdf_pattern/BDF_PATTERN

File: tools/extra/fpgadiag/common.py (hunk 5 above)
Comment (tswhison) [inline tools/extra/fpgadiag/common.py]: isn't it a convention that module-level globals be capitalized?

File: tools/extra/fpgadiag/common.py (hunk 6 above)
Comment (tswhison) [inline tools/extra/fpgadiag/common.py]: ... module-level globals should be capitalized.

File: tools/extra/fpgadiag/common.py (hunk 1 above)
Comment (tswhison) [inline tools/extra/fpgadiag/common.py]: ... module-level globals should be capitalized.

File: tools/extra/fpgadiag/common.py (hunk 4 above)
Comment (tswhison) [inline tools/extra/fpgadiag/common.py]: s/Invlid/Invalid/

File: tools/extra/fpgadiag/fpgastats.py [hunk 8]
```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env python
-# Copyright(c) 2019, Intel Corporation
+# Copyright(c) 2018, Intel Corporation
```
Comment (jelonanX) [inline tools/extra/fpgadiag/fpgastats.py]: s/2018/2019

Comment (tswhison) [inline tools/extra/fpgadiag/fpgastats.py]: s/2018/2018-2019/

File: tools/extra/fpgadiag/mactest.py [hunk 9]
```diff
@@ -30,20 +30,30 @@
...
 from common import FpgaFinder, exception_quit
-from common import COMMON, hexint
+from common import COMMON, convert_argument_str2hex
 
-SYS_IF = '/sys/class/net'
-DIVIDE = '-' * 80
-HOST_SIDE = 1
+sys_if = '/sys/class/net'
+divide = '-' * 80
+FVL_SIDE = 1
```
Comment (jelonanX) [inline tools/extra/fpgadiag/mactest.py]: inconsistent use of upper and lower cases with macros

File: tools/extra/fpgadiag/mactest.py [hunk 10]
```diff
@@ -30,20 +30,30 @@
...
 import sys
+import glob
 from common import FpgaFinder, exception_quit
-from common import COMMON, hexint
+from common import COMMON, convert_argument_str2hex
 
-SYS_IF = '/sys/class/net'
-DIVIDE = '-' * 80
-HOST_SIDE = 1
+sys_if = '/sys/class/net'
```
Comment (tswhison) [inline tools/extra/fpgadiag/mactest.py]: ... module-level globals should be capitalized.

File: tools/extra/fpgadiag/mactest.py [hunk 11]
```diff
@@ -30,20 +30,30 @@
...
+divide = '-' * 80
```
Comment (tswhison) [inline tools/extra/fpgadiag/mactest.py]: ... module-level globals should be capitalized.

**Pull request merged**